anvil/src/cmd.rs

```diff
@@ -442,6 +443,10 @@ pub struct AnvilEvmArgs {
     /// Enable steps tracing used for debug calls returning geth-style traces
     #[clap(long, visible_alias = "tracing")]
     pub steps_tracing: bool,
+
+    /// Enable autoImpersonate on startup
+    #[clap(long, short, visible_alias = "unlocked")]
+    pub unlocked: bool,
```

Comment:
--unlocked already has a slightly different meaning in cast, 
perhaps we rename this to `--auto-impersonate` instead?